tests/qunit/core/editor/document/save/commands/draft.spec.js

```diff
@@ -16,6 +16,9 @@ export const Draft = () => {
 			const testStatuses = [ 'publish', 'private' ],
 				document = elementor.documents.getCurrent();
 
+			// Since draft: will reject if 'document.editor.isChanged == false'.
+			$e.internal( 'document/save/set-is-modified', { status: true } );
```

Comment:
Make the changes below via command, so it will set it as modified.
(line 23 `document.container.settings.set( 'post_status', status );` )

Reply:
Handled: https://github.com/elementor/elementor/pull/11884/commits/4be961b321ebe29ded67e2d39a97891e4fb60dba